app/src/layouts/appLayout/footer/footer.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import { FormattedMessage } from 'react-intl';
+import styles from './footer.scss';
+
+const cx = classNames.bind(styles);
+
+export const Footer = () => (
+  <div className={cx('footer')}>
+    <ul className={cx('footer-links')}>
+      <li>
+        <a
+          href="https://github.com/reportportal/reportportal"
+          target="_blank"
+        >
+          <FormattedMessage
+            id={'Footer.git'}
+            defaultMessage={'Fork us on GitHub'}
+          />
+        </a>
+      </li>
+      <li>
+        <a
+          href="https://reportportal-slack-auto.herokuapp.com/"
```

Comment:
To define hrefs use constants described in src/common/utils/referenceDictionary.js file.